modules/tracking/src/trackerBoosting.cpp

```diff
@@ -126,7 +126,7 @@ bool TrackerBoostingImpl::initImpl( const Mat& image, const Rect2d& boundingBox
   Mat_<int> intImage;
   Mat_<double> intSqImage;
   Mat image_;
-  cvtColor( image, image_, CV_RGB2GRAY );
+  cvtColor( image, image_, COLOR_BGR2GRAY );
```

Comment:
(RFC) should i keep it `COLOR_RGB2GRAY`

Reply:
Yes, lets avoid using of legacy "_c.h" headers and their symbols.

Reply:
R-G-B & B-G-R

Reply:
Oops...
Looks good to me.
OpenCV default format for I/O is `BGR`
(except Android - RGBA - they should explicitly pass gray images for such algorithms).